packages/@okta/vuepress-site/docs/reference/rate-limits/index.md

```diff
@@ -45,6 +45,6 @@ If an org-wide rate limit is exceeded, an HTTP 429 status code is returned. You
 
 * [Previous rate limits](/docs/reference/rl-previous/): This content covers the rate limits for orgs that were created before 2018-05-17.
 
-* [Okta API endpoints and per-user limits](/docs/reference/rl-additional-limits/#okta-api-endpoints-and-per-user-limits): API endpoints that take username and password credentials, including the [Authentication API](/docs/reference/api/authn/) and the [OAuth 2.0 resource owner password flow](/docs/guides/implement-password/), have a per-username rate limit to prevent brute force attacks with the user's password.
+* [Okta API endpoints and per-user limits](/docs/reference/rl-additional-limits/#okta-api-endpoints-and-per-user-limits): API endpoints that take username and password credentials, including the [Authentication API](/docs/reference/api/authn/) and the [OAuth 2.0 resource owner password flow](/docs/guides/implement-password/), have a per-username rate limit to prevent brute force attacks with the user's password. [SMS and Call factor endpoints](/docs/reference/api/factors/#factor-lifecycle-operations) also have a per-username rate limit.
```

Comment:
Should this link to https://developer.okta.com/docs/reference/rl-additional-limits/#per-user-limits (or the new section that we will add for SMS and Voice RLs in rl-additional-limits) instead.